src/jit/importer.cpp

```diff
@@ -6462,6 +6463,11 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
         call->gtCall.callSig  = new (this, CMK_CorSig) CORINFO_SIG_INFO;
         *call->gtCall.callSig = calliSig;
 #endif // DEBUG
+
+        if (IsTargetAbi(CORINFO_CORERT_ABI))
```

Comment:
@fadimounir If this is only something that is applicable to your private branches, let's skip this for now. This needs to be specced out. For example, we definitely don't want this for all callvirts - just the generic ones - this might require a JitInterface change, etc.